### PR TITLE
Adapt OpenShift Code Sample to Hazelcast 4.0

### PR DESCRIPTION
**Pull request by leszko**
I'll send one more PR for the Kubernetes Code Sample update (CC: @mmedenjak)

### CODE REVIEW

File: hazelcast-integration/openshift/client-apps/ocp-demo-frontend/src/main/resources/hazelcast-client.xml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <hazelcast-client xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
                   xsi:schemaLocation="http://www.hazelcast.com/schema/client-config
-                               http://www.hazelcast.com/schema/client-config/hazelcast-client-config-3.11.xsd"
+                               http://www.hazelcast.com/schema/client-config/hazelcast-client-config-4.0.xsd"
```
Comment (mesutcelik) [inline hazelcast-integration/openshift/client-apps/ocp-demo-frontend/src/main/resources/hazelcast-client.xml]: What about replacing this file by hazelcast-client.yaml then we do not need maintain xsd versions.

File: hazelcast-integration/openshift/hazelcast-cluster/hazelcast-enterprise-openshift-centos/hazelcast.yaml [hunk 2]
```diff
@@ -12,7 +12,7 @@ labels:
 parameters:
 - name: HAZELCAST_IMAGE
   description: "Defines the location of Hazelcast image"
-  value: hazelcast/hazelcast-enterprise:3.12
```
Comment (mesutcelik) [inline hazelcast-integration/openshift/hazelcast-cluster/hazelcast-enterprise-openshift-centos/hazelcast.yaml]: This image is alpine based but we call this file centos. I don't think we should keep centos named folder anymore.

File: hazelcast-integration/openshift/hazelcast-cluster/hazelcast-openshift-origin/hazelcast.yaml [hunk 3]
```diff
@@ -12,7 +12,7 @@ labels:
 parameters:
 - name: HAZELCAST_IMAGE
   description: "Defines the location of Hazelcast image"
-  value: hazelcast/hazelcast:3.12
```
Comment (mesutcelik) [inline hazelcast-integration/openshift/hazelcast-cluster/hazelcast-openshift-origin/hazelcast.yaml]: Openshift Origin is renamed so we can either update this folder or remove completely.

File: hazelcast-integration/openshift/hazelcast-cluster/hazelcast-enterprise-openshift-centos/hazelcast.yaml (hunk 2 above)
Comment (leszko) [reply hazelcast-integration/openshift/hazelcast-cluster/hazelcast-enterprise-openshift-centos/hazelcast.yaml]: done

File: hazelcast-integration/openshift/hazelcast-cluster/hazelcast-openshift-origin/hazelcast.yaml (hunk 3 above)
Comment (leszko) [reply hazelcast-integration/openshift/hazelcast-cluster/hazelcast-openshift-origin/hazelcast.yaml]: done

File: hazelcast-integration/openshift/client-apps/ocp-demo-frontend/src/main/resources/hazelcast-client.xml (hunk 1 above)
Comment (leszko) [reply hazelcast-integration/openshift/client-apps/ocp-demo-frontend/src/main/resources/hazelcast-client.xml]: ok, done

**Comment by leszko** [comment]
@mesutcelik thanks for the comments. Addressed them all. PTAL

**Pull request merged**